lib/ukfallocbuddy/fallocbuddy.c

```diff
@@ -1451,6 +1465,10 @@ static int bfa_do_addmem(struct buddy_framealloc *bfa, void *metadata,
 	bfa_zone_add(bfa, zone);
 	bfa->fa.total_memory += len;
 
+#ifdef CONFIG_LIBUKFALLOC_STATS
+	uk_falloc_stats_global.total_memory += len;
+#endif /* CONFIG_LIBUKFALLOC_STATS */
```

Comment:
@michpappas are you sure this is okay? Total memory never decreases (like `free_memory`)

Reply:
I think so. The API provides an `addmem` operation but no ability to remove memory. The `alloc` and `free` functions operate on previously added memory.